src/it/scala/org/apache/spark/sql/SFDateFunctionsSuite.scala

```diff
@@ -0,0 +1,925 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql
+
+import java.sql.{Date, Timestamp}
+import java.text.SimpleDateFormat
+import java.time.{Instant, LocalDateTime, ZoneId}
+import java.util.concurrent.TimeUnit
+import java.util.{Locale, TimeZone}
+
+import org.apache.spark.SparkUpgradeException
+import org.apache.spark.sql.catalyst.util.DateTimeTestUtils.{CEST, LA}
+import org.apache.spark.sql.catalyst.util.DateTimeUtils
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.internal.SQLConf
+import org.apache.spark.sql.snowflake.{SFQueryTest, SFTestSessionBase}
+import org.apache.spark.sql.types.DoubleType
+import org.apache.spark.unsafe.types.CalendarInterval
+
+class SFDateFunctionsSuite extends SFQueryTest with SFTestSessionBase {
```

Comment:
Is this test suite new developed or ported from SNOW-PARK? If it is new, I think they should be put in net.snowflake...

Reply:
It's inherited. Also, it extends directly from some of the Spark test classes.